### DISCUSSION TITLE
bug in version 0.38.3

### DISCUSSION THREAD

**Discussion by gogi100** [rejetto/hfs#144] [General]
the hfs hase a bug, when i create a folder. i click create folder, i type folder name, then the folder is created successful and i go to folder, but i receive message not found, the pictures are below
![createfolderbutton](https://user-images.githubusercontent.com/21057856/224498694-629c5ee2-b3d7-45bf-8ef6-e86ec9dafb93.jpg)
![folder name](https://user-images.githubusercontent.com/21057856/224498699-85f5eb1d-af43-479b-b158-d7baec23eee2.jpg)
![go to folder](https://user-images.githubusercontent.com/21057856/224498702-6daa80cc-c4ff-4e46-af46-73a3fbc32b24.jpg)
![nofolder](https://user-images.githubusercontent.com/21057856/224498708-a82b4de5-5d4a-4e37-bf9e-827ac2a460ec.jpg)

**Comment by gogi100** [comment]
i back in folder deljeni and this folder 15 exists.. the problem is with path of folder or breadcrumb

**Comment by rejetto** [comment]
1. so you create a folder named just "15" ?
2. in what folder are you when you create it? it's not visible in your screenshots.
3. is there any folder named "deljeni15" ?

**Reply by gogi100** [reply]
i created folder '15', in folder 'deljeni'. the folder 'deljeni15' does not exists

**Comment by rejetto** [comment]
i'm making several tests, but all works good here, I don't know how to make it happen.
is it possible for you to make a video instead of screenshots?

**Reply by gogi100** [reply]
i created video. the first login on the hfs, and i create folder, it's exists the problem, when i back in parent folder and i go to created folder, the creating folders works.

https://user-images.githubusercontent.com/21057856/224504236-0e3a0045-edda-4028-a6e9-4774395400c6.mov

**Comment by rejetto** [comment]
thanks, the video seems useful because i see that "deljeni" is a folder you get redirect automatically at login, and you didn't mention that before.
I think that's the key of the problem.
You can probably solve by using final slash: deljeni/
But I will now study the problem so that it can work also without /

**Reply by gogi100** [reply]
i set for that user 'test' redirect to the folder 'deljeni'

**Reply by rejetto** [reply]
exactly,
set "deljeni/" instead, so you don't have to wait for next version to solve

**Comment by gogi100** [comment]
i find one more bug. when i try to delete file, the selected file is not delete. i refresh page and i try one more, the deleting works.

https://user-images.githubusercontent.com/21057856/224505594-e429d4b8-d5c8-4d27-a1c1-b96466ae94a4.mov

**Reply by rejetto** [reply]
I think it's happening because you didn't apply the solution I suggested you about "redirect" with final slash.
I see that in first part of your vide you don't have final slash in the address.

**Reply by gogi100** [reply]
Ok, i will try it

**Reply by gogi100** [reply]
it's works for the both cases, thanks

**Comment by rejetto** [comment]
i see that you have a "hole" in your buttons bar, because you hid the zip button.
<img width="243" alt="image" src="https://user-images.githubusercontent.com/1367199/224539401-40d84301-0ddd-40c4-9d6c-0ae86c062713.png">

To hide the zip button without getting a blank space, you should use this style:
`#menu-bar>a { display: none; }`

**Comment by rejetto** [comment]
I noticed you have errors in your language file, causing numbers to be displayed without text.
<img width="138" alt="image" src="https://user-images.githubusercontent.com/1367199/224539246-fae15378-4250-4146-967c-2708b303c3db.png">

It's because you translated also commands, like "plural".
I tried to fix the problems in your json file. See the fixed version at
https://github.com/rejetto/hfs/blob/main/langs/hfs-lang-sr.json

**Reply by gogi100** [reply]
I fixed i my json file
